Makefile

```diff
@@ -5,7 +5,6 @@ PREFIX = /usr/local
 SHAREPREFIX = ${PREFIX}/share/lighthouse
 DOLLAR = $$
 
-CC=gcc
```

Comment:
something like `CC?=cc` would be more robust, not everyone has CC in their env

Reply:
Fixed that, thank you!